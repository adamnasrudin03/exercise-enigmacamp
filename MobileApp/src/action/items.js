import {
  SAVE_ITEM_REQUEST,
  DELETE_ITEM_REQUEST,
  FIND_ITEM_REQUEST,
  FIND_ITEMS_REQUEST,
} from './constants';

export function save(data) {
  return {
    type: SAVE_ITEM_REQUEST,
    data: data,
  };
}

export function deleteById(id) {
  return {
    type: DELETE_ITEM_REQUEST,
    id: id,
  };
}

export function findById(id) {
  return {
    type: FIND_ITEM_REQUEST,
    id: id,
  };
}

export function findAllItems(params) {
  return {
    type: FIND_ITEMS_REQUEST,
    params: params,
  };
}
