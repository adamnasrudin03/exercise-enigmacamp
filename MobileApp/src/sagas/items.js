import {
  SAVE_ITEM_REQUEST,
  SAVE_ITEM_SUCCESS,
  SAVE_ITEM_FAILURE,
  DELETE_ITEM_REQUEST,
  DELETE_ITEM_SUCCESS,
  DELETE_ITEM_FAILURE,
  FIND_ITEM_REQUEST,
  FIND_ITEM_SUCCESS,
  FIND_ITEM_FAILURE,
  FIND_ITEMS_REQUEST,
  FIND_ITEMS_SUCCESS,
  FIND_ITEMS_FAILURE,
} from './../action/constants';
import {commonAxios} from '../utils/apiUtils';
import {put, takeLatest} from 'redux-saga/effects';

function* findAll(action) {
  const {search = {}, sort = 'asc', page = 0, size = 10} = action.params || {};
  try {
    const data = yield commonAxios.get('items', {
      params: {...search, sort, page, size},
    });

    yield put({
      type: FIND_ITEMS_SUCCESS,
      data: data,
    });
  } catch (error) {
    yield put({
      type: FIND_ITEMS_FAILURE,
      error: error,
    });
  }
}
function* findById(action) {
  try {
    const data = yield commonAxios.get(`items/${action.id}`);

    yield put({
      type: FIND_ITEM_SUCCESS,
      data: data,
    });
  } catch (error) {
    yield put({
      type: FIND_ITEM_FAILURE,
      error: error,
    });
  }
}

function* deleteById(action) {
  try {
    const data = yield commonAxios.delete(`items/${action.id}`);

    yield put({
      type: DELETE_ITEM_SUCCESS,
      data: data,
    });
  } catch (error) {
    yield put({
      type: DELETE_ITEM_FAILURE,
      error: error,
    });
  }
}
function* save(action) {
  const {id, name} = action.data;
  try {
    const data = yield id
      ? commonAxios.put(`items/${id}`, {id, name})
      : commonAxios.post(`items`, {name});

    yield put({
      type: SAVE_ITEM_SUCCESS,
      data: data,
    });
  } catch (error) {
    yield put({
      type: SAVE_ITEM_FAILURE,
      error: error,
    });
  }
}

export function* watchItems() {
  yield takeLatest(FIND_ITEMS_REQUEST, findAll);
}
export function* watchItem() {
  yield takeLatest(FIND_ITEM_REQUEST, findById);
}
export function* watchDeleteItem() {
  yield takeLatest(DELETE_ITEM_REQUEST, deleteById);
}
export function* watchSaveItem() {
  yield takeLatest(SAVE_ITEM_REQUEST, save);
}
