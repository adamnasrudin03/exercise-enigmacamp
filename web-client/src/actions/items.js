import {
  SAVE_ITEM_REQUEST,
  SAVE_ITEM_SUCCESS,
  SAVE_ITEM_FAILURE,
  DELETE_ITEM_REQUEST,
  DELETE_ITEM_SUCCESS,
  DELETE_ITEM_FAILURE,
  FIND_ITEM_REQUEST,
  FIND_ITEM_SUCCESS,
  FIND_ITEM_FAILURE,
  FIND_ITEMS_REQUEST,
  FIND_ITEMS_SUCCESS,
  FIND_ITEMS_FAILURE,
} from "./constant";

import { commonAxios } from "../utils/apiUtils";

function deleteItemSucces(data) {
  return {
    type: DELETE_ITEM_SUCCESS,
    data: data,
  };
}
function deleteItemFailure(error) {
  return {
    type: DELETE_ITEM_FAILURE,
    error: error,
  };
}

export const deleteById = (id) => (dispatch) => {
  dispatch({
    type: DELETE_ITEM_REQUEST,
  });

  commonAxios
    .delete(`items/${id}`)
    .then((data) => {
      dispatch(deleteItemSucces(data));
    })
    .catch((error) => {
      console.log(error);
      dispatch(deleteItemFailure(error));
    });
};

function saveItemSucces(data) {
  return {
    type: SAVE_ITEM_SUCCESS,
    data: data,
  };
}
function saveItemFailure(error) {
  return {
    type: SAVE_ITEM_FAILURE,
    error: error,
  };
}

export const save = ({ id, name } = {}) => (dispatch) => {
  dispatch({
    type: SAVE_ITEM_REQUEST,
  });
  const request = id
    ? commonAxios.put(`items/${id}`, { id, name })
    : commonAxios.post(`items`, { name });

  request
    .then((data) => {
      dispatch(saveItemSucces(data));
    })
    .catch((error) => {
      console.log(error);
      dispatch(saveItemFailure(error));
    });
};

function findItemSucces(data) {
  return {
    type: FIND_ITEM_SUCCESS,
    data: data,
  };
}
function findItemFailure(error) {
  return {
    type: FIND_ITEM_FAILURE,
    error: error,
  };
}
export const findById = (id) => (dispatch) => {
  dispatch({
    type: FIND_ITEM_REQUEST,
  });

  commonAxios
    .get(`items/${id}`)
    .then((data) => {
      dispatch(findItemSucces(data));
    })
    .catch((error) => {
      console.log(error);
      dispatch(findItemFailure(error));
    });
};

function findItemsSucces(data) {
  return {
    type: FIND_ITEMS_SUCCESS,
    data: data,
  };
}
function findItemsFailure(error) {
  return {
    type: FIND_ITEMS_FAILURE,
    error: error,
  };
}
export const findAll = ({
  search = {},
  sort = "asc",
  page = 0,
  size = 10,
} = {}) => (dispatch) => {
  dispatch({
    type: FIND_ITEMS_REQUEST,
  });

  commonAxios
    .get("items", {
      params: { ...search, sort, page, size },
    })
    .then((data) => {
      dispatch(findItemsSucces(data));
    })
    .catch((error) => {
      dispatch(findItemsFailure(error));
    });
};
