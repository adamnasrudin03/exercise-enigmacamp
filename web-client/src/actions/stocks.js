import {
    SAVE_STOCK_REQUEST,
    SAVE_STOCK_SUCCESS,
    SAVE_STOCK_FAILURE,
    DELETE_STOCK_REQUEST,
    DELETE_STOCK_SUCCESS,
    DELETE_STOCK_FAILURE,
    FIND_STOCK_REQUEST,
    FIND_STOCK_SUCCESS,
    FIND_STOCK_FAILURE,
    FIND_STOCKS_REQUEST,
    FIND_STOCKS_SUCCESS,
    FIND_STOCKS_FAILURE,
    FIND_STOCKS_SUMMARY_REQUEST,
    FIND_STOCKS_SUMMARY_SUCCESS,
    FIND_STOCKS_SUMMARY_FAILURE
} from './constant';

import { commonAxios } from '../utils/apiUtils'




function saveStockSucces(data) {
    return {
        type: SAVE_STOCK_SUCCESS,
        data: data
    }
};
function saveStockFailure(error) {
    return {
        type: SAVE_STOCK_FAILURE,
        error: error
    }
};

export const save = ({ id, item, quantity, unit } = {}) =>
    (dispatch) => {
        dispatch({
            type: SAVE_STOCK_REQUEST
        });
        const request = id ?
            commonAxios.put(`stocks/${id}`, { id, item, quantity, unit }) :
            commonAxios.post(`stocks`, { item: { id: item.id }, quantity, unit: { id: unit.id } });

        request
            .then(data => {
                dispatch(saveStockSucces(data));
            })
            .catch(error => {
                console.log(error);
                dispatch(saveStockFailure(error));
            });
    };

export const deleteById = (id) =>
    (dispatch) => {
        dispatch({
            type: DELETE_STOCK_REQUEST
        });

        commonAxios.delete(`stocks/${id}`)
            .then(data => data)
            .then(data => {
                dispatch(deleteStockSucces(data));
            })
            .catch(error => {
                console.log(error);
                dispatch(deleteStockFailure(error));
            });
    };
function deleteStockSucces(data) {
    return {
        type: DELETE_STOCK_SUCCESS,
        data: data
    }
};
function deleteStockFailure(error) {
    return {
        type: DELETE_STOCK_FAILURE,
        error: error
    }
};


export const findById = (id) =>
    (dispatch) => {
        dispatch({
            type: FIND_STOCK_REQUEST
        });
        commonAxios.get(`stocks/${id}`)
            .then(data => {
                dispatch(findStockSucces(data));
            })
            .catch(error => {
                console.log(error);
                dispatch(findStockFailure(error));
            });
    };

function findStockSucces(data) {
    return {
        type: FIND_STOCK_SUCCESS,
        data: data
    }
};
function findStockFailure(error) {
    return {
        type: FIND_STOCK_FAILURE,
        error: error
    }
};

export const findAll = ({ sort = 'asc', page = 0, size = 10 } = {}) =>
    (dispatch) => {
        dispatch({
            type: FIND_STOCKS_REQUEST
        });

        commonAxios.get('stocks', {
            params: { sort, page, size }
        })
            .then(data => {
                dispatch(findStocksSucces(data));
            })
            .catch(error => {
                dispatch(findStocksFailure(error));
            });
    };

function findStocksSucces(data) {
    return {
        type: FIND_STOCKS_SUCCESS,
        data: data
    }
};
function findStocksFailure(error) {
    return {
        type: FIND_STOCKS_FAILURE,
        error: error
    }
};



export const findAllSummary = ({  sort = 'asc'} = {}) =>
    (dispatch) => {
        dispatch({
            type: FIND_STOCKS_SUMMARY_REQUEST
        });

        commonAxios.get('stocks/summary', {
            params: { sort }
        })
            .then(data => {
                dispatch(findSummarySucces(data));
            })
            .catch(error => {
                dispatch(findSummaryFailure(error));
            });
    };

function findSummarySucces(data) {
    return {
        type: FIND_STOCKS_SUMMARY_SUCCESS,
        data: data
    }
};
function findSummaryFailure(error) {
    return {
        type: FIND_STOCKS_SUMMARY_FAILURE,
        error: error
    }
};
