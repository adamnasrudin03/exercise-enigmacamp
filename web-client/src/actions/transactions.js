import {
  SAVE_TRANSACTION_REQUEST,
  SAVE_TRANSACTION_SUCCESS,
  SAVE_TRANSACTION_FAILURE,
  DELETE_TRANSACTION_REQUEST,
  DELETE_TRANSACTION_SUCCESS,
  DELETE_TRANSACTION_FAILURE,
  FIND_TRANSACTION_REQUEST,
  FIND_TRANSACTION_SUCCESS,
  FIND_TRANSACTION_FAILURE,
  FIND_TRANSACTIONS_REQUEST,
  FIND_TRANSACTIONS_SUCCESS,
  FIND_TRANSACTIONS_FAILURE,
  FIND_TRANSACTIONS_SUMMARY_REQUEST,
  FIND_TRANSACTIONS_SUMMARY_SUCCESS,
  FIND_TRANSACTIONS_SUMMARY_FAILURE,
} from "./constant";

import { commonAxios } from "../utils/apiUtils";

function deleteTransactionSucces(data) {
  return {
    type: DELETE_TRANSACTION_SUCCESS,
    data: data,
  };
}
function deleteTransactionFailure(error) {
  return {
    type: DELETE_TRANSACTION_FAILURE,
    error: error,
  };
}

export const deleteById = (id) => (dispatch) => {
  dispatch({
    type: DELETE_TRANSACTION_REQUEST,
  });

  commonAxios
    .delete(`transactions/${id}`)
    .then((data) => {
      dispatch(deleteTransactionSucces(data));
    })
    .catch((error) => {
      console.log(error);
      dispatch(deleteTransactionFailure(error));
    });
};

function saveTransactionSucces(data) {
  return {
    type: SAVE_TRANSACTION_SUCCESS,
    data: data,
  };
}
function saveTransactionFailure(error) {
  return {
    type: SAVE_TRANSACTION_FAILURE,
    error: error,
  };
}

export const save = ({ id, amount, type, description } = {}) => (dispatch) => {
  dispatch({
    type: SAVE_TRANSACTION_REQUEST,
  });

  const request = id
    ? commonAxios.put(`transactions/${id}`, { id, amount, type, description })
    : commonAxios.post(`transactions`, { amount, type, description });

  request
    .then((data) => data)
    .then((data) => {
      dispatch(saveTransactionSucces(data));
    })
    .catch((error) => {
      console.log(error);
      dispatch(saveTransactionFailure(error));
    });
};

function findTransactionSucces(data) {
  return {
    type: FIND_TRANSACTION_SUCCESS,
    data: data,
  };
}
function findTransactionFailure(error) {
  return {
    type: FIND_TRANSACTION_FAILURE,
    error: error,
  };
}
export const findById = (id) => (dispatch) => {
  dispatch({
    type: FIND_TRANSACTION_REQUEST,
  });

  commonAxios
    .get(`transactions/${id}`)
    .then((data) => {
      dispatch(findTransactionSucces(data));
    })
    .catch((error) => {
      console.log(error);
      dispatch(findTransactionFailure(error));
    });
};

function findTransactionsSucces(data) {
  return {
    type: FIND_TRANSACTIONS_SUCCESS,
    data: data,
  };
}
function findTransactionsFailure(error) {
  return {
    type: FIND_TRANSACTIONS_FAILURE,
    error: error,
  };
}
export const findAll = ({
  search = {},
  sort = "asc",
  page = 0,
  size = 10,
} = {}) => (dispatch) => {
  dispatch({
    type: FIND_TRANSACTIONS_REQUEST,
  });

  commonAxios
    .get("transactions", {
      params: { ...search, sort, page, size },
    })
    .then((data) => {
      dispatch(findTransactionsSucces(data));
    })
    .catch((error) => {
      dispatch(findTransactionsFailure(error));
    });
};

function findTransactionsSummarySucces(data) {
  return {
    type: FIND_TRANSACTIONS_SUMMARY_SUCCESS,
    data: data,
  };
}
function findTransactionsSummaryFailure(error) {
  return {
    type: FIND_TRANSACTIONS_SUMMARY_FAILURE,
    error: error,
  };
}
export const findAllSummary = ({ sort = "asc" } = {}) => (dispatch) => {
  dispatch({
    type: FIND_TRANSACTIONS_SUMMARY_REQUEST,
  });

  commonAxios
    .get("transactions/summary", {
      params: { sort },
    })
    .then((data) => {
      dispatch(findTransactionsSummarySucces(data));
    })
    .catch((error) => {
      dispatch(findTransactionsSummaryFailure(error));
    });
};
