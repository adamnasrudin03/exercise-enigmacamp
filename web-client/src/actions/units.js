import {
    SAVE_UNIT_REQUEST,
    SAVE_UNIT_SUCCESS,
    SAVE_UNIT_FAILURE,
    DELETE_UNIT_REQUEST,
    DELETE_UNIT_SUCCESS,
    DELETE_UNIT_FAILURE,
    FIND_UNIT_REQUEST,
    FIND_UNIT_SUCCESS,
    FIND_UNIT_FAILURE,
    FIND_UNITS_REQUEST,
    FIND_UNITS_SUCCESS,
    FIND_UNITS_FAILURE
} from './constant';

import { commonAxios } from '../utils/apiUtils'


function deleteUnitSucces(data) {
    return {
        type: DELETE_UNIT_SUCCESS,
        data: data
    }
};
function deleteUnitFailure(error) {
    return {
        type: DELETE_UNIT_FAILURE,
        error: error
    }
};

export const deleteById = (id) =>
    (dispatch) => {
        dispatch({
            type: DELETE_UNIT_REQUEST
        });

        commonAxios.delete(`units/${id}`)
            .then(data => {
                dispatch(deleteUnitSucces(data));
            })
            .catch(error => {
                console.log(error);
                dispatch(deleteUnitFailure(error));
            });
    };





function saveUnitSucces(data) {
    return {
        type: SAVE_UNIT_SUCCESS,
        data: data
    }
};
function saveUnitFailure(error) {
    return {
        type: SAVE_UNIT_FAILURE,
        error: error
    }
};

export const save = ({ id, name, description } = {}) =>
    (dispatch) => {
        dispatch({
            type: SAVE_UNIT_REQUEST
        });
        const request = id ?
            commonAxios.put(`units/${id}`, { id, name, description }) :
            commonAxios.post(`units`, { name, description });

        request
            .then(data => {
                dispatch(saveUnitSucces(data));
            })
            .catch(error => {
                console.log(error);
                dispatch(saveUnitFailure(error));
            });
    };





function findUnitSucces(data) {
    return {
        type: FIND_UNIT_SUCCESS,
        data: data
    }
};
function findUnitFailure(error) {
    return {
        type: FIND_UNIT_FAILURE,
        error: error
    }
};
export const findById = (id) =>
    (dispatch) => {
        dispatch({
            type: FIND_UNIT_REQUEST
        });

        commonAxios.get(`units/${id}`)
            .then(data => {
                dispatch(findUnitSucces(data));
            })
            .catch(error => {
                console.log(error);
                dispatch(findUnitFailure(error));
            });
    };





function findUnitsSucces(data) {
    return {
        type: FIND_UNITS_SUCCESS,
        data: data
    }
};
function findUnitsFailure(error) {
    return {
        type: FIND_UNITS_FAILURE,
        error: error
    }
};
export const findAll = ({ search = {}, sort = 'asc', page = 0, size = 10 } = {}) =>
    (dispatch) => {
        dispatch({
            type: FIND_UNITS_REQUEST
        });

        commonAxios.get('units', {
            params: { ...search, sort, page, size }
        })
            .then(data => {
                dispatch(findUnitsSucces(data));
            })
            .catch(error => {
                dispatch(findUnitsFailure(error));
            });
    };


