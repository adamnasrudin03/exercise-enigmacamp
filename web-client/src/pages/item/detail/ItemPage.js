import React from "react";
import styles from "./styles";
import { withStyles } from "@material-ui/core/styles";
import Page from "../../../components/Page";
import { Save as SaveIcon, ArrowBackIos as BackIcon } from "@material-ui/icons";
import {
    Button,
    TextField,
    CircularProgress,
    Backdrop
} from "@material-ui/core";
import { save, findById } from "../../../actions/items";
import { connect } from "react-redux";
import Alert from '@material-ui/lab/Alert';
import { Snackbar } from '@material-ui/core';


class ItemPage extends React.Component {
    constructor(props) {
        super(props);

        const { match } = this.props;

        this.state = {
            form: {
                id: match.params.id,
                name: ""
            },
            alertShow: false,
            errorMessage: "",
            error: {}
        };
    }

    componentDidMount() {
        const { form } = this.state;
        if (form.id) {
            this.props.findById(form.id);
        }
    }

    componentDidUpdate(prevProps, prevState) {
        const { data, error, saveError, saveData, history } = this.props;

        if (prevProps.data !== data) {
            this.setState({ form: data });
        } else if (prevProps.error !== error) {
            this.setState({ error: error });
        } else if (error && prevProps.saveError !== saveError) {
            this.setState({ error: saveError });
        } else if (saveData && prevProps.saveData !== saveData) {
            history.push('/items');
        }
    }

    onChange = event => {
        const { name, value } = event.target;
        const { form } = this.state;

        this.setState({ form: { ...form, [name]: value } });
    };

    hideAlert = () => {
        this.setState({ alertShow: false })
    }

    onSubmit = event => {
        event.preventDefault();
        const { form } = this.state;
        if (form.name === "" ) {
            this.setState({
                alertShow: true,
                errorMessage: " name is not empty "
            });

        } else if (form.name.length <  3) {
            this.setState({
                alertShow: true,
                errorMessage: " name is minimum 3 character "
            });
        }else {
            this.props.save(this.state.form);
        }
    };

    render() {
        const { classes, loading, saveError } = this.props;
        const { form, error } = this.state;
        const errorData = saveError?.data || {};
        return (
            <Page error={error}>
                {!loading ?

                    <form noValidate autoComplete="off" onSubmit={this.onSubmit}>
                        {form.id && (
                            <div className={classes.formField}>
                                <TextField
                                    id="id"
                                    name="id"
                                    label="ID"
                                    value={form.id}
                                    fullWidth
                                    inputProps={{ readOnly: true }}
                                />
                            </div>
                        )}
                        <div className={classes.formField}>
                            <TextField
                                id="name"
                                name="name"
                                label="Name"
                                value={form.name}
                                fullWidth
                                error={errorData.name}
                                helperText={errorData.name ? errorData.name[0] : null}
                                onChange={this.onChange}
                            />
                        </div>

                        <Snackbar open={this.state.alertShow}
                            autoHideDuration={6000}
                            onClose={this.hideAlert}>
                            <Alert onClose={this.hideAlert}
                                elevation={6} variant="filled"
                                severity="error">
                                {this.state.errorMessage}
                            </Alert>
                        </Snackbar>

                        <div >
                            <Button
                                className={classes.formBack}
                                variant="contained" color="secondary" href="/web-client/items"
                                disabled={loading}
                                startIcon={<BackIcon />}
                            >
                                Back
                              </Button>
                            <Button
                                className={classes.formButton}
                                variant="contained" color="primary"
                                type="submit" disabled={loading}
                                startIcon={<SaveIcon />}
                            >
                                Save
                              </Button>
                        </div>
                    </form> :
                    <Backdrop
                        className={classes.backdrop}
                        open={true}
                    >
                        <CircularProgress color="primary" />
                    </Backdrop>
                }
            </Page>
        );
    }
}

const mapStateToProps = state => ({
    saveData: state.saveItem.data,
    saveError: state.saveItem.error,
    data: state.findItemById.data,
    loading: state.findItemById.loading || state.saveItem.loading,
    error: state.findItemById.error
});

const mapDispatchToProps = {
    save,
    findById
};

export default withStyles(styles, { withTheme: true })(
    connect(mapStateToProps, mapDispatchToProps)(ItemPage)
);