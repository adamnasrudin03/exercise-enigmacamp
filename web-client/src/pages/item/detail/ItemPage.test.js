import React from 'react';
import { mount } from 'enzyme';
import ItemPage from './ItemPage';
import * as itemActions from '../../../actions/items';
import { MemoryRouter as Router } from 'react-router-dom';



const data = {
  id: 1,
  name: "krupuk"
};

itemActions.findById = jest.fn().mockReturnValue(data);

describe('ItemPage', () => {
  let wrapper;

  beforeAll(() => {
    wrapper = mount(
      <Router>
        <ItemPage match={{ params: { id: 1 } }} />
      </Router>
    );
  });


  it('should have data', () => {
    const component = wrapper.find("ItemPage");
    console.log(component.state('id'));
    expect(component.state('id')).toEqual(data.id);
    console.log(component.state('name'));
    expect(component.state('name')).toEqual(data.name);
   

  });

});