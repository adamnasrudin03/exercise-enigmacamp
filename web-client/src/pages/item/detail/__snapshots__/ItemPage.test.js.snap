// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemPage should match snapshot 1`] = `undefined`;
